docs/build/program/apis/robot.md

```diff
@@ -248,8 +248,8 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 
 ### WithChannel
 
-Create a robot that is connected to a machine over the given channel.
-Any robots created using this method will NOT automatically close the channel upon exit.
+Create a machine that is connected to a machine over the given channel.
```

Comment:
If we refer to it as robot client in AtAddress we should also use that phrasing here, both methods create RobotClients. Fine w/ referring to it as robot client or machine

Reply:
Good point.
```suggestion
Create a RobotClient that is connected to a machine over the given channel.
```